### DISCUSSION TITLE
A very dear request that will improve this mod by 1000%

### DISCUSSION THREAD

**Discussion by Alexsys** [zer0Kerbal/Pteron#23] [General]
The mini shuttle is grand, but there is one issue that makes it worse than grand, and its the landing gear.
It has a terrible invisible ghost hitbox that is very sensitive and badly placed, causing take offs and landings to be broken and explosive 99% of the times, creative workarounds don't work

Firestorm mod sadly is VERY junky on modern KSP and damages many mods without fix, it can be concluded that firestorm is one of those mods that if you install you gotta pass on way too many good modern others, making firestorm obsolete. 
Also maybe its even incompatible with 1.12? unsure.

So the issue,
The model hitbox or frame includes in itself those invisible wheels even if not deployed, and that is the massive destroyer explained why below:

If there would be an option to have the Pteron wings **detached** in the editor, from their current state which are Wings including gears in them, that would be amazing.

This picture displays why
https://i.imgur.com/0BXSaWf.jpeg

It means nothing if they're in or out, its just a decoration apparently, if you simply use stock game and pteron at least.
https://i.imgur.com/HXz9FGE.jpeg

Employing all the powers of Tweakscale to try and make wheels that overshadow the invisible explosive hitbox still doesn't eliminate the fact that it forces explosions evrey single time the wheels take stress (from the weight of the craft), forcing them to go up n down, thus the invisible STATIC hitbox gets touched and blows up.

https://i.imgur.com/40WWyp5.jpeg
(this still blows up, and this is my finest attempt yet after so many tries ever)

So sadly in its current state, other than a space drone and a glider with parachutes to return to kerbin, the craft's purpose is stripped from it by a large portion,
you either gotta give up the wings or give up making it land or take off from atmosphere ground.

pteron is unlandable on kerbin, and un take offable.

valve please fix

**Comment by zer0Kerbal** [comment]
@Alexsys thank you.
- I've landed it numerous times. It is landable. 
- Any modeling issues are beyond my ability. 
- Firespitter is unusable, and both issues and a PR have been sitting with the repo for quite some time. It is the reason I just made them an animation.  With @lisias's version of Firespitter - they work, but the vessel spawns 1000m above the launchpad/runway.
- (https://github.com/snjo/Firespitter/issues)
- could try this one: (https://github.com/net-lisias-kspu/Firespitter)
- It is meant to launch as the shuttle did - as a rocket. hence skids.
- will see what else can be done upon the next pass.

**Reply by Lisias** [reply]
The 1.000m jump is worked around with World Stabiliser.

I was working on a definitive fix for it, the same way I fixed the Instant Crafts Annihilations due overheat on some parts on launch, but with KSP Community Fixes around, I choose to wait.

I you ask my opinion, KSP (and some of the most important add'ons) is being **terribly** maintained nowadays. It's the reason I forked Firespitter for myself.

**Reply by zer0Kerbal** [reply]
@Lisias 
> World Stabilizer  

so installing say (https://github.com/net-lisias-kspu/WorldStabilizer) _should_ fix the spawning issue (1000m)?

**Reply by Lisias** [reply]
Yes. At least, it works for me - you may had found/created a new use case where World Stabiliser borks, but if this is the case, I will be motivated to fix it on KSP-Recall.

Once the fix is implemented on Recall, you can bet your mouse it will be implemented too on KSP Community Fix. Weird enough, win-win situation. :)

**Reply by Alexsys** [reply]
I believe I've tried this path in the past, I've had firespitter and world stabilizer but then it messed up with bon voyage mod that couldnt take it to account when spawning a vehicle after "theoretical travelling", and the hasstle it brought to many other mods was just not worth it, so i gave it up.

i did manage to get the wheels on the pteron just barely suicideablelanding right, for it to land without exploding by the ghost wheels, but that took way too long.

**Reply by Lisias** [reply]
So apparently we have a good reason to fix the thing on KSP-Recall so...

**Reply by zer0Kerbal** [reply]
> wheels

- I am holding back releases/working on a bunch of high profile mods because Firespitter is mucked up (and or KSP) - and I do not know enough about the stock wheels system to convert them to stock (which would work).
- there are another bunch of fixes that are needed for Pteron - including getting the FX working again on the hydra/other engines, and now the customized parachutes.

**Reply by zer0Kerbal** [reply]
and I certainly prefer discussing this here rather on the forums. 🍌
